### PR TITLE
Board add methods

### PR DESCRIPTION
**Pull request by Serg-Bartosh**

### CODE REVIEW

**Review by alena-bartosh — DISMISSED** [review]
Good enough

File: src/board.py [hunk 1]
```diff
@@ -12,3 +12,30 @@ def __init__(self, height: int, weight: int) -> None:
...
+        return self.__cells[point.x][point.y] is False
+
+    def get_cells(self) -> CellsType:
+        return self.__cells
+
+    def is_cells_empty(self, points: list[Point]) -> bool:
+        for point in points:
+            if self.is_cell_empty(point) is False:
+                return False
+        return True
```
Comment (s373r) [inline src/board.py:34]: add a blank line before

File: src/board.py [hunk 2]
```diff
@@ -12,3 +12,30 @@ def __init__(self, height: int, weight: int) -> None:
...
+
+    def set_cells_with_value(self, points: list[Point], value: bool) -> None:
+        for point in points:
+
```
Comment (s373r) [inline src/board.py]: remove an extra blank line

**Review by s373r — CHANGES_REQUESTED** [review]

File: src/board.py (hunk 2 above)
Comment (Serg-Bartosh) [reply src/board.py]: Done

File: src/board.py (hunk 1 above)
Comment (Serg-Bartosh) [reply src/board.py:34]: Done

**Review by alena-bartosh — APPROVED** [review]

**Review by s373r — APPROVED** [review]

**Pull request merged**